foreman/requirements.txt

```diff
@@ -9,7 +9,7 @@ billiard==3.5.0.2         # via celery
 boto==2.46.1
 celery==4.0.2
 django==1.11.1
-GEOparse==0.1.10
+geoparse==0.1.10
```

Comment:
Oh good point. Seems annoying that they can't maintain Python's convention of lowercase package names. However I really should just remove this because I actually am not using it any more.